File: exercises/to-other/fizzBuzz/fizzBuzz.js
function fizzBuzz() {
    var arr = []
    for (let i = 0; i < 101; i++) {
        arr.push(i)
    }
    var newArray = arr
    for (let i in arr) {
        if ((arr[i] % 3 === 0) && (arr[i] % 5 === 0)) {
            newArray[i] = "fizzbuzz"
        } else if (arr[i] % 5 === 0) {
            newArray[i] = "buzz"
        } else if (arr[i] % 3 === 0) {
            newArray[i] = "fizz"
        } else {
            newArray[i] = arr[i]
        }
    }

    return newArray;
}

console.log(fizzBuzz())

File: exercises/to-other/fizzBuzz/fizzBuzz.go
package main

import (
	"fmt"
)

func main() {
	for i := 1; i < 101; i++ {
		if i % 15 == 0 {
			fmt.Println("fizzBuzz")
		} else if i % 5 == 0 {
			fmt.Println("buzz")
		} else if i % 3 == 0 {
			fmt.Println("fizz")
		} else {
			fmt.Println(i)
		}
	}
}
